entity-service-api/src/main/proto/org/hypertrace/entity/data/service/v1/entity_data_request.proto

```diff
@@ -54,3 +57,13 @@ message AttributeFilter {
   AttributeValue attributeValue = 3;
   repeated AttributeFilter childFilter = 4;
 }
+
+message OrderByExpression {
```

Comment:
Since this is being used in EQS as well, do you want to extract into a separate file?

Reply:
I prefer to keep the API separate with different object, we don't want to mix them since the use case would very be different. The code repetition worth keeping the 2 API separate